src/cljs/proton/layers/core/keybindings.cljs

```diff
@@ -142,9 +142,11 @@
                   :title "focus right pane"}
               :m {:action "maximize-panes:maximize"
                   :title "maximize pane"}
+              :n {:action "application:new-window"
+                  :title "new window"}
               :o {:title "close others"
                   :fx panes/close-other-panes}
-              :u {:action "open last-closed window"
```

Comment:
Oh! Looks like this was a oversight in previous versions as well: `:action` should be `pane:reopen-closed-item` and `:title` should be `open last-closed pane`. Could you update this? 

Also please give it a try if this is actually the right command for re-opening a closed pane.